packages/api/src/platforms/vtex/clients/commerce/index.ts

```diff
@@ -670,6 +670,24 @@ export const VtexCommerce = (
           {}
         )
       },
+      getUserGrantedResources: ({
+        userId,
+        resourceKey,
+      }: {
+        userId: string
+        resourceKey: string
+      }): Promise<boolean> => {
+        const headers: HeadersInit = withAutCookie(forwardedHost, account)
+
+        return fetchAPI(
+          `${base}/api/license-manager/storefront/users/${userId}/resources/${resourceKey}/granted`,
```

Comment:
I've been testing, and I think we should prevent calling this request when the `userId` or `resourceKey` is `undefined`. Scenario: when there is no logged user.

<img width="1025" height="419" alt="Screenshot 2025-09-15 at 12 59 52" src="https://github.com/user-attachments/assets/0e217940-93c5-455b-af7f-a58ce29378c9" />